Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean

```diff
@@ -795,6 +796,44 @@ theorem sin_pi_div_three : sin (π / 3) = √3 / 2 := by
   congr
   ring
 
+theorem four_mul_sq_cos_pi_div_five_sub_two_mul_sub_one_eq_zero :
```

Comment:
I really don't like how long this name is. I think it's fine to deviate from usual naming standards for something like this, how about `cos_pi_div_five_equation` or `cos_pi_div_five_polynomial`?

Reply:
I am happy to report that this horrible name is gone. I took Jireh's suggested `quadratic_root_cos_pi_div_five`.